src/js/signin/components/SearchHelpSignIn.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import _ from 'lodash';
+
+import HelpMenu from '../../common/components/HelpMenu';
+import SearchMenu from '../../common/components/SearchMenu';
+import SignInProfileMenu from './SignInProfileMenu';
+
+import { updateLoggedInStatus, toggleSearchHelpUserMenu } from '../actions';
+
+class SearchHelpSignIn extends React.Component {
+  render() {
+    let content;
+    const login = this.props.login;
```

Comment:
Minor improvement could be made here; destructuring the `this.props.login` to pull out `currentlyLoggedIn` and `utilitiesMenuIsOpen`. Also destructuring `profile` out of `this.props`.